### ISSUE TITLE
Number of devices on the same network

### ISSUE THREAD

**Issue by pvint** [vintlabs/fauxmoESP#113]
**[Original report](https://bitbucket.org/xoseperez/fauxmoesp/issue/113) by Roger Espinoza (Bitbucket: [Roger Espinoza](https://bitbucket.org/Roger Espinoza), ).**

----------------------------------------

Hello, excuse me, I would like to know if it is possible to connect more than one \(physical\) esp device on the same network, what happens is that after adding an esp device in the alexa app, the following does not recognize me, thank you very much in advance.

**Comment by rogerespinoza** [comment]
Hello, there is no news?

**Comment by pvint** [comment]
Hi Roger,

I have used more than one physical device often in the past, and I just tested with two ESP32 devices, each with 8 fauxmo "devices", and when I do "Alexa Discover" it finds all 16 new devices, and they work fine. (Tested using a Gen3 Echo Dot).

If you're still having trouble, try setting:
`#define DEBUG_FAUXMO                Serial`
in your code and see if you se the following when you do "Alexa discover":
`[FAUXMO] Responding to M-SEARCH request`

**Comment by rogerespinoza** [comment]
Hello pvint, you will see my problem occurs because if I already connect a physical device (esp8266), the next one that I try to connect alexa does not recognize it, I suppose it is due to the port that the library already has assigned to work, and when more than one on the same network is no longer detected, how do you do to avoid problems at that point?

**Comment by pvint** [comment]
It's perfectly ok to use the same port on different devices (that's actually the standard way). 

One thing to verify: Are you using a different device name when creating the devices (in `fauxmo.addDevice`  on the second one? 


Also, check in the alexa app that there aren't "stale" devices in the list.

**Comment by rogerespinoza** [comment]
I will verify all these points and report to you, thanks for the help.

**Comment by Tejas-MD** [comment]
Hi, Good to hear someone's maintaining this library @pvint Good Luck! , it really needs a little work for what its worth. 

  About this issue: I tested with a single esp32 for almost a day or two and it worked pretty fine.. But when I added another today, It discovered all my devices and works too but....  My esp32 resets.(This happens almost at a defined interval after entering the  `void loop()` . 

 I have set port to 80 but just saw about the higher bandwidth option.  Will try that too and let you know. 

Will also attach the debug output and reset cause from the serial monitor if that bandwidth thing doesn't work.
